src/o365/teams/commands/report/teams-report-useractivityuserdetail.ts

```diff
@@ -31,8 +31,8 @@ class TeamsReportUserActivityUserDetailCommand extends GraphCommand {
 
   public getTelemetryProperties(args: CommandArgs): any {
     const telemetryProps: any = super.getTelemetryProperties(args);
-    telemetryProps.period = typeof args.options.period !== 'undefined';
-    telemetryProps.date = typeof args.options.date !== 'undefined';
+    telemetryProps.period = args.options.period;
+    telemetryProps.date = args.options.date;
```

Comment:
Tracking the exact date doesn't add any value to us, because we can't aggregate it in any meaningful way. Instead, we should track if it was specified.